bedrock/products/templates/products/vpn/platforms/android.html

```diff
@@ -0,0 +1,167 @@
+    {% endcall %}
+
+    {% call vpn_platform_content_well(
+      title=ftl('vpn-platforms-shared-what-youll-get'),
+      image_url='img/products/vpn/platforms/vpn-cntwell-check-mark.png',
+      class='vpn-content-well-left-half'
+    ) %}
+      <ul class="mzp-u-list-styled">
+        <li>{{ ftl('vpn-platforms-shared-feature-item-protection', devices=vpn_devices) }}</li>
+        <li>{{ ftl('vpn-platforms-shared-feature-item-access', countries=vpn_countries) }}</li>
+        <li>{{ ftl('vpn-platforms-shared-feature-item-encryption') }}</li>
+        <li>{{ ftl('vpn-platforms-shared-feature-item-bandwidth') }}</li>
+        <li>{{ ftl('vpn-platforms-shared-feature-item-logs') }}</li>
+      </ul>
+    {% endcall %}
+
+    {% call vpn_platform_content_well(
+      title=ftl('vpn-android-wifi-headline'),
+      image_url='img/products/vpn/platforms/vpn-cntwell-public-wifi-mobile.png',
+      class='vpn-content-well-right-half'
+    ) %}
+      <p>{{ ftl('vpn-android-wifi-copy') }}</p>
+    {% endcall %}
+
+    {% call vpn_platform_content_well(
+      title=ftl('vpn-android-privacy-headline'),
+      image_url='img/products/vpn/platforms/vpn-cntwell-one-click.png',
+      class='vpn-content-well-left-half'
+    ) %}
+      <p>{{ ftl('vpn-android-privacy-copy') }}</p>
+    {% endcall %}
+
+    {% call vpn_platform_content_well(
+      title=ftl('vpn-android-servers-headline', servers=vpn_servers, countries=vpn_countries),
+      image_url='img/products/vpn/platforms/vpn-cntwell-globe.png',
+      class='vpn-content-well-right-half'
+    ) %}
+      <p>{{ ftl('vpn-android-servers-copy', countries=vpn_countries) }}</p>
+    {% endcall %}
+
+    {% call vpn_platform_content_well(
+      title=ftl('vpn-android-devices-headline', devices=vpn_devices),
+      image_url='img/products/vpn/platforms/vpn-cntwell-devices.png',
+      class='vpn-content-well-left-half'
+    ) %}
+      <p>{{ ftl('vpn-android-devices-copy') }}</p>
+    {% endcall %}
+
+    {% call vpn_platform_content_well(
+      title=ftl('vpn-android-speed-headline'),
+      image_url='img/products/vpn/platforms/vpn-cntwell-speed-guage.png',
+      class='vpn-content-well-right-half'
+    ) %}
+      <p>{{ ftl('vpn-android-speed-copy') }}</p>
+    {% endcall %}
+
+    {% call vpn_platform_content_well(
+      title=ftl('vpn-android-log-headline'),
+      image_url='img/products/vpn/platforms/vpn-cntwell-privacy.png',
+      class='vpn-content-well-left-half'
+    ) %}
+      <p>{{ ftl('vpn-android-log-copy') }}</p>
+    {% endcall %}
+
+    <!-- Custom content well for the CTA -->
+    <section class="vpn-content-well-cta">
+      <div class="vpn-content-well-cta-container">
+        <h2 class="vpn-content-well-title">{{ ftl('vpn-platforms-shared-cta-headline') }}</h2>
+        <div class="vpn-content-well-desc">
+          <a class="vpn-button" href="{{ url('products.vpn.landing') }}">{{ ftl('vpn-platforms-shared-cta') }}</a>
+          <div class="vpn-hero-guarantee">
```

Comment:
Same question as above: should this get the style treatment from the main page? And this one could also be a `p` instead of a div.